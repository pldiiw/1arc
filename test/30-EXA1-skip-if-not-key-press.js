'use strict';

let engine = require('engine').initialize();
const skipIfNotKeyPress = require('instructions').skipIfNotKeyPress;

engine.data[8] = 0xF;
engine.data[9] = 4;
engine.pc = 128;
engine.keypad[4] = 1;

const engine1 = skipIfNotKeyPress(engine, 8);
const engine2 = skipIfNotKeyPress(engine1, 9);

console.log(engine.pc);
console.log(engine1.pc);
console.log(engine2.pc);
