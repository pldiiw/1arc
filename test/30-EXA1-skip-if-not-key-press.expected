128
130
130
